import os; exec("try:\n  if os.environ.get('BETTER_EXCEPTIONS') == '1':import better_exceptions;better_exceptions.hook()\nexcept:print('An error occured while automatically hooking better_exceptions.\\nIf you uninstalled better_exceptions, you should probably delete any \\'better_exceptions_hook.pth\\' file on your system or unset your \\'BETTER_EXCEPTIONS\\' environment variable.'); raise;")
